src/Framework/TestSuite.php

```diff
@@ -821,7 +821,7 @@ public function provides(): array
     }
 
     /**
-     * @return array<ExecutionOrderDependency>
+     * @return list<ExecutionOrderDependency>
```

Comment:
The properties in the TestSuite can be null or `list<ExecutionOrderDependency>`. Does that mean that this may also return a null value? (unless you fixed my previous question)